### DISCUSSION TITLE
Docs improvement

### DISCUSSION THREAD

**Discussion by fergdev** [Tlaster/PreCompose#171] [General]
I was just getting started with ViewModel and Koin Precompose for Compose-Multi-Platform project and could find no where in the docs to include your root library. 

Maybe you could add this to documentation? 

pre-compose = {module = "moe.tlaster:precompose", version.ref = "pre-compose" }

Or am I bad at reading repos? haha :)

Thanks for making this!

**Comment by Tlaster** [comment]
The [setup guide](https://github.com/Tlaster/PreCompose/blob/master/docs/setup.md) should tell you about how to setup the dependencies. Through the docs needs an update since as the development there're things that being added but not in the docs.

**Reply by fergdev** [reply]
Legend, got things working. Thanks again for working on this Library.
